tests/unit_tests/shared/test_feature_toggling.py

```diff
@@ -0,0 +1,69 @@
+import os
+from unittest.mock import Mock
+
+import pytest
+
+import ert.__main__
+from ert.__main__ import ert_parser
+from ert.cli import TEST_RUN_MODE
+from ert.shared.feature_toggling import FeatureToggling
+
+
+@pytest.fixture(autouse=True)
+def mocked_valid_file(monkeypatch):
+    monkeypatch.setattr(ert.__main__, "valid_file", Mock(return_value=""))
```

Comment:
What's the purpose of this?

Reply:
ert_parser tries to validate that the ert config file exists. there is little point in making a temp file for this, so I override the test instead.